src/main/java/lotto/model/enums/MatchCount.java

```diff
@@ -0,0 +1,33 @@
+package lotto.model.enums;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum MatchCount {
+    ZERO(0, 0), ONE(1, 0), TWO(2, 0), THREE(3, 5000),
```

Comment:
```suggestion
    ZERO(0, 0),
    ONE(1, 0),
    TWO(2, 0),
    THREE(3, 5000),
    FOUR(4, 50000),
    FIVE(5, 1500000),
    SIX(6, 2000000000);
```
이렇게 나열해주는게 한 눈에 더 잘 보이지 않을까요?

Reply:
넵!